floodmodeller_api/units/logical_rules.py

```diff
@@ -0,0 +1,76 @@
+"""
+Flood Modeller Python API
+Copyright (C) 2023 Jacobs U.K. Limited
+
+This program is free software: you can redistribute it and/or modify it under the terms of the GNU General Public License 
+as published by the Free Software Foundation, either version 3 of the License, or (at your option) any later version.
+
+This program is distributed in the hope that it will be useful, but WITHOUT ANY WARRANTY; without even the implied warranty 
+of MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU General Public License for more details. 
+
+You should have received a copy of the GNU General Public License along with this program.  If not, see https://www.gnu.org/licenses/.
+
+If you have any query about this program or this License, please contact us at support@floodmodeller.com or write to the following 
+address: Jacobs UK Limited, Flood Modeller, Cottons Centre, Cottons Lane, London, SE1 2QG, United Kingdom.
+"""
+
+import pandas as pd
+
+from .helpers import join_10_char, join_n_char_ljust
+from ..diff import check_item_with_dataframe_equal
+
+### Variables, Rules & Varrules Classes ###
+
+
+class Variables:
```

Comment:
Could you also add _get_diff and __eq__ methods to this class (look at initial conditions for example)